optuna/storages/rdb/storage.py

```diff
@@ -602,3 +596,170 @@ def __del__(self):
 
         if hasattr(self, 'scoped_session'):
             self.remove_session()
+
+    def upgrade(self):
+        # type: () -> None
+        """Upgrade the storage schema."""
+
+        self._version_manager._upgrade()
+
+    def get_current_version(self):
+        # type: () -> str
+        """Return the schema version currently used by this storage."""
+
+        return self._version_manager._get_current_version()
+
+    def get_head_version(self):
+        # type: () -> str
+        """Return the latest schema version."""
+
+        return self._version_manager._get_head_version()
+
+    def get_all_versions(self):
+        # type: () -> List[str]
+        """Return the schema version list."""
+
+        return self._version_manager._get_all_versions()
+
+
+class _VersionManager(object):
+    def __init__(self, url, engine, scoped_session):
+        # type: (str, Engine, orm.Session) -> None
```

Comment:
The type of `scoped_session` is not `Session` but `scoped_session`.

Reply:
Oops..., sorry for my mistake. 
I corrected the type at the commit https://github.com/pfnet/optuna/pull/336/commits/5029171128c5798bc280084e73992a3ebf188545.